### ISSUE TITLE
Gene Sequence does not show up in Feature page

### ISSUE THREAD

**Issue by martacsilva** [tripal/tripal#998]
Hi!
I am currently trying to display the sequence of my genes in their feature page. My Manage Display currently looks like this: 
![Screenshot from 2019-09-26 12-34-47](https://user-images.githubusercontent.com/47527249/65685607-4755e200-e05a-11e9-95b9-79b56d9d53e4.png)

The Sequence Coordinates display just fine but not anything else. I've tried moving to other groups and changing from Hidden to Inline and Above.
What am I missing?

**Comment by bradfordcondon** [comment]
Unfortunately this is not a bug but a known issue.
you might want to look at this issue:
https://github.com/tripal/tripal/issues/732

Currently with Tripal, if you want gene sequences to display on the gene page, you need to associate sequences directly with the gene features, as [the sequence field only looks in that features->residues field](https://github.com/tripal/tripal/blob/e4d4c57fa53bb0d9d8807b3e99ee84c77769bc63/tripal_chado/includes/TripalFields/data__sequence/data__sequence.inc#L74-L178).


you can try the `data__sequence_features` field available as part of the [manage analyses module](https://github.com/statonlab/tripal_manage_analyses), which does try to display sequences that are **derived** from alignments, but it only works for **children** (theres a [writeup here](https://github.com/statonlab/tripal_manage_analyses/blob/master/docs/gene_fields.rst).

So in short

- base tripal wants the sequence to live on the feature model.  So, if you import a GFF but only load FASTA for protein and mrna, then sequences will only be available on the protein and mrna feature pages.
- You can get sequences on your gene page by loading in FASTA files for your gene features
- You can try `tripal_manage_analyses` but it may or may not work for your needs.

**Comment by martacsilva** [comment]
Thanks for the quick answer! 
I am almost sure that we did load a FASTA for our gene features but I will confirm that and get back to you. But in case we didn't, simply loading it should fix my problem without any further issues, right?

**Comment by martacsilva** [comment]
Also I do have the `tripal_manage_analyses` module and it's great, but it's not exactly what we were looking for :(

**Comment by bradfordcondon** [comment]
if the residues are directly associated with the feature type you are looking at, they should load.  Please double check thats the case and if its not we'll figure it out

**Comment by martacsilva** [comment]
Hi again @bradfordcondon !
Can you give an example of the kind of FASTA file you are mentioning?

**Comment by bradfordcondon** [comment]
In the [Tripal tutorial](https://tripal.readthedocs.io/en/latest/user_guide/example_genomics/genomes_genes.html#loading-feature-data) there are two FASTA files: one for the scaffolds and one for the mRNA.
- Citrus sinensis-scaffold00001.fasta
- Citrus sinensis-orange1.1g015632m.g.fasta

If you follow this guide, you get sequences on your mRNA and scaffolds.  You don't get seuqences on your gene pages (and if you look at the last image in the guide we can see there are no sequences).  If you wanted sequences on your gene pages, you'd need to upload a FASTA file with sequences for the genes.  

So, if you wanted gene sequences, you'd supply a FASTA file with entries for the genes instead of the mRNAs/scaffolds, and load it pointed at the gene feature type.  

The first record, `>orange1.1g015632m PAC:18136217 (mRNA) Citrus sinensis` is for the mRNA.  if i look at the GFF, i can see that the gene name is  `orange1.1g015632m.g` so my fasta file would be something like 
```
>orange1.1g015632m.g

ATGATATTATAGGGCC .....
```

**Comment by martacsilva** [comment]
I loaded a FASTA file with the format of:
>unique_name
sequence...

And still no sequence shows up in my gene page. Is there something else I can try?

EDIT: It did work! But not for all features? One of the genes still has no sequence and that was the one I was looking at.

**Comment by martacsilva** [comment]
All good! All my genes (or at least the ones I've looked at) have sequences!

Thank you so much @bradfordcondon !
